docs/app/documentation/component-docs/date-picker/examples/date-picker-form-example.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component } from '@angular/core';
+import { FormGroup, FormControl } from '@angular/forms';
+
+@Component({
+    selector: 'fd-date-picker-form-example',
+    template: `
+        <form [formGroup]="customForm">
+            <fd-date-picker formControlName="date"></fd-date-picker>
+        </form>
+
+        Selected Date: {{ customForm.controls.date.value.date | date: 'fullDate' }}
```

Comment:
This should be: `Selected Date: {{ customForm.controls.date.value.date ? customForm.controls.date.value.date : 'null' }}`